### PR TITLE
[DOC] kafka documentation

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: docs/UnderstandingDaiNetworkListenerConfigurations.md [hunk 1]
```diff
@@ -173,6 +173,22 @@ The name field of this type is always `rabbitmq`.
...
+...
+"name": "kafka",
+"arguments": {
+"bootstrap.servers": "kafka_hosted_machine:9092",
+"group.id": "boot_states",
+"schema.registry.url": "http://kafka_hosted_machine:8081",
+"auto.commit.enable": false,
+"auto.offset.reset": "earliest",
+"specific.avro.reader": "true",
+"topics": "stateChanges"
```
Comment (paul-amonson) [inline docs/UnderstandingDaiNetworkListenerConfigurations.md:188]: Indent is missing?

File: docs/WLMAdapterForeignBusConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## Example: ##
+```json
+{
+"bootstrap.servers": "kafka_hosted_machine:9092",
+"group.id": "respective_subscribed_group_id",
+"schema.registry.url": "http://kafka_hosted_machine:8081",
+"auto.commit.enable": "true",
+"auto.offset.reset": "earliest",
+"specific.avro.reader": "true",
+"topics": "respective_subscribed_topic"
```
Comment (paul-amonson) [inline docs/WLMAdapterForeignBusConfiguration.md:11]: Indentation is missing?

**Review by paul-amonson — DISMISSED** [review]

File: docs/WLMAdapterForeignBusConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Identifies the consumer group of a given consumer. group_id of the topic names of kafka bus
+
+### schema.registry.url ###
+Registered schema URL the provider is expecting message to send to. URL of the registered schemas of kafka bus
+
+### auto.commit.enable ###
+Consumer's offset will be periodically committed in the background. By default, it is set to false
+
+### auto.offset.reset ###
+Offset from which next new record will be fetched. Used ONLY if consumer don't have a valid offset committed.
```
Comment (nbwhite) [inline docs/WLMAdapterForeignBusConfiguration.md:29]: Should be "...if consumer **doesn't** have a valid offset..."

Comment (sindhu-106) [reply docs/WLMAdapterForeignBusConfiguration.md]: updated changes. Thanks Nathan

**Review by nbwhite — APPROVED** [review]

**Review by paul-amonson — APPROVED** [review]

**Pull request merged**